### PR TITLE
8322545: Declare newInsets as static in ThemeReader.cpp

### PR DESCRIPTION
**Pull request by rajamah**
Since newInsets() function is a helper function used in ThemeReader.cpp only , this change declares it as static.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8322545](https://bugs.openjdk.org/browse/JDK-8322545): Declare newInsets as static in ThemeReader.cpp (**Bug** - P4)

### Reviewers
 * [Sergey Bylokhov](https://openjdk.org/census#serb) (@mrserb - **Reviewer**) ⚠️ Review applies to [a19c14f2](https://git.openjdk.org/jdk/pull/17310/files/a19c14f2c816f46a83c83bbe4bd3622106ada7d5)
 * [Alexey Ivanov](https://openjdk.org/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/17310/head:pull/17310` \
`$ git checkout pull/17310`

Update a local copy of the PR: \
`$ git checkout pull/17310` \
`$ git pull https://git.openjdk.org/jdk.git pull/17310/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17310`

View PR using the GUI difftool: \
`$ git pr show -t 17310`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/17310.diff">https://git.openjdk.org/jdk/pull/17310.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/17310#issuecomment-1881648470)

### CODE REVIEW

**Review by mrserb — APPROVED** [review]

File: src/java.desktop/windows/native/libawt/windows/ThemeReader.cpp
Comment (aivanov-jdk) [inline src/java.desktop/windows/native/libawt/windows/ThemeReader.cpp:1]: Update the copyright year to 2024.

**Review by aivanov-jdk — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**